Mathlib/RingTheory/Spectrum/Prime/Topology.lean

```diff
@@ -589,6 +590,12 @@ lemma range_comap_algebraMap_localization_compl_eq_range_comap_quotientMk
   rw [range_comap_of_surjective _ _ surj, localization_away_comap_range _ (C c)]
   simp [Polynomial.ker_mapRingHom, Ideal.map_span]
 
+lemma isRetrocompact_iff {U : Set (PrimeSpectrum R)} (hU : IsOpen U) :
```

Comment:
What property is there for `PrimeSpectrum` that makes this true and not for more general topological spaces?

Reply:
It's the presence of a basis consisting of compact sets whose pairwise intersections are compact too. See https://stacks.math.columbia.edu/tag/0069

Note that the general case is already proved: it's `IsTopologicalBasis.isRetrocompact_iff_isCompact` and we're using it here

Reply:
Actually, it's equivalent to there being a basis of compact sets and the space being quasi-separated. See #21325.

Reply:
See #21362